shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/context/CommonBeans.java

```diff
@@ -110,6 +111,16 @@ HiveConf replicaHiveConf(
     return hiveConf;
   }
 
+  @Bean
+  HiveConf sourceHiveConf(Configuration baseConfiguration, SourceCatalog sourceCatalog) {
```

Comment:
Is it fine that there are now two `HiveConf` beans? I guess spring knows which one to inject by the variable/method name?

Reply:
Yeah. We need two HiveConf, one each for source and target. It resolves by the method name I think.